### ISSUE TITLE
Errors in translation

### ISSUE THREAD

**Issue by clacesarano** [mysociety/dirittodisapere-theme#97]
Some users reported us some mistakes in the translation of the subject of the mail and in other parts of the website. I have updated the translations and completed all the untranslated lines.
Can you update the translations and check that
{{site_name}} = Chiedi
{{law_used_short}} = accesso civico
{{law_used_full}}= Decreto Trasparenza (D.lgs. 33/2013)

**Comment by lizconlan** [comment]
ToDo:
 - [ ] check/update the site name in the config
 - [ ] check/fix the `law_used` text in the site code

**Comment by lizconlan** [comment]
`SITE_NAME` was set to "Diritto di Sapere" has been updated to "Chiedi", will update on next deploy

**Comment by lizconlan** [comment]
Transifex updates would require a site upgrade, holding off on this for now as that's a lengthy process

**Comment by clacesarano** [comment]
How lengthy?
on the 23rd of December there is a new ATI law in force therefore on that day we will have to update the help pages and update the green banner announcing the change.
Maybe we should program a unique ugrade to go online on that day (if it fits your schedule)?

**Comment by lizconlan** [comment]
> How lengthy?

Something like 3 days' work, I think

**Comment by garethrees** [comment]
Done what we can at this point once #100 is merged. I've made a new ticket for upgrading to 0.25 (https://github.com/mysociety/dirittodisapere-theme/issues/103).

**Comment by lizconlan** [comment]
Site update with the `law_used` translations. Closing in favour of #103
